introduction.md

```diff
@@ -0,0 +1 @@
+Introduction: My name is John and I am working on my Git skills!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.